content/guides/core-concepts/cypress-studio.md

```diff
@@ -2,17 +2,9 @@
 title: Cypress Studio
 ---
```

Comment:
You can add `e2eSpecific: true` to the front matter at the top of the file, like [this](https://github.com/cypress-io/cypress-documentation/blob/master/content/guides/end-to-end-testing/auth0-authentication.md)

There's logic on the header to pick this up and add the pill next to the header, above the horizontal rule so it's a "page level" concept.